src/mcp/tools/storage/get_storage_rules.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { getLatestRulesetName, getRulesetContent } from "../../../gcp/rules";
+
+export const get_storage_rules = tool(
+  {
+    name: "get_storage_rules",
+    description: "Retrieves the Firebase Cloud Storage Rules for the default bucket.",
+    inputSchema: z.object({}),
```

Comment:
We may want to provide an argument for storage bucket name in the future.

There can be >1 Firebase Storage in a project.

Reply:
Left a TODO